config/dev/{% if use_ingestion and ingestion_framework == 'airbyte' %}airbyte.yml{% endif %}.tmpl

```diff
@@ -0,0 +1,110 @@
+airbyte_connection_id: airbyte_connection_id
+airbyte_url: https://airbyte.dataops-dev.getindata.dev
+connections:
+  POSTGRES_BQ_CONNECTION:
+    destinationId: [[ destination_id ]]
+    name: POSTGRES_BQ_CONNECTION
+    namespaceDefinition: customformat
+    namespaceFormat: [[ dataset ]]
+    operationIds: []
+    sourceId: [[ source_id ]]
+    status: active
+    operations:
+    - name: Normalization
+      operatorConfiguration:
+        normalization:
+          option: basic
+        operatorType: normalization
+      workspaceId: [[ workspace_id ]]
+    syncCatalog:
+      streams:
+      - config:
+          aliasName: raw_orders
+          cursorField: []
+          destinationSyncMode: append
+          primaryKey: []
+          selected: true
+          syncMode: full_refresh
+        stream:
+          defaultCursorField: []
+          jsonSchema:
+            properties:
+              id:
+                airbyte_type: integer
+                type: number
+              order_date:
+                format: date
+                type: string
+              status:
+                type: string
+              user_id:
+                airbyte_type: integer
+                type: number
+            type: object
+          name: raw_orders
+          namespace: public
+          sourceDefinedPrimaryKey: []
+          supportedSyncModes:
+          - full_refresh
+      - config:
+          aliasName: raw_payments
+          cursorField: []
+          destinationSyncMode: append
+          primaryKey: []
+          selected: true
+          syncMode: full_refresh
+        stream:
+          defaultCursorField: []
+          jsonSchema:
+            properties:
+              amount:
+                airbyte_type: integer
+                type: number
+              id:
+                airbyte_type: integer
+                type: number
+              order_id:
+                airbyte_type: integer
+                type: number
+              payment_method:
+                type: string
+            type: object
+          name: raw_payments
+          namespace: public
+          sourceDefinedPrimaryKey: []
+          supportedSyncModes:
+          - full_refresh
+      - config:
+          aliasName: raw_customers
+          cursorField: []
+          destinationSyncMode: append
+          primaryKey: []
+          selected: true
+          syncMode: full_refresh
+        stream:
+          defaultCursorField: []
+          jsonSchema:
+            properties:
+              first_name:
+                type: string
+              id:
+                airbyte_type: integer
+                type: number
+              last_name:
+                type: string
+            type: object
+          name: raw_customers
+          namespace: public
+          sourceDefinedPrimaryKey: []
+          supportedSyncModes:
+          - full_refresh
+destinations: {}
+sources: {}
+tasks:
```

Comment:
what these tasks are about?

Reply:
These task are for composer to create ingestion task before applying dbt transformations
https://dbt-airflow-factory.readthedocs.io/en/latest/features.html#adding-ingestion-layer

Reply:
These task are for composer to create ingestion task before applying dbt transformations
https://dbt-airflow-factory.readthedocs.io/en/latest/features.html#adding-ingestion-layer